### ISSUE TITLE
Apple privacy manifest duplication issue

### ISSUE THREAD

**Issue by DanielPritchard** [auth0/react-native-auth0#893]
### Checklist

- [x] The issue can be reproduced in the [react-native-auth0 sample app](https://github.com/auth0-samples/auth0-react-native-sample) (or N/A).
- [X] I have looked into the [Readme](https://github.com/auth0/react-native-auth0#readme), [Examples](https://github.com/auth0/react-native-auth0/blob/master/EXAMPLES.md), and [FAQ](https://github.com/auth0/react-native-auth0/blob/master/FAQ.md) and have not found a suitable solution or answer.
- [X] I have looked into the [API documentation](https://auth0.github.io/react-native-auth0/) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/react-native-auth0/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

Upgrading to package version 3.2.0 has caused an error at build time, it appears my local `PrivacyInfo.xcprivacy` is conflicting with this package, I get the following error

```
error: Multiple commands produce '/Users/dan/Library/Developer/Xcode/DerivedData/myproject-ebjsfdssgrakgmbywhgztfeqpxtd/Build/Products/Debug-iphoneos/myproject.development.local.app/PrivacyInfo.xcprivacy'
    note: Target 'myproject.development.local' (project 'myproject') has copy command from '/Users/dan/projects/github/repo/apps/myproject/ios/PrivacyInfo.xcprivacy' to '/Users/dan/Library/Developer/Xcode/DerivedData/myproject-ebjsfdssgrakgmbywhgztfeqpxtd/Build/Products/Debug-iphoneos/myproject.development.local.app/PrivacyInfo.xcprivacy'
    note: That command depends on command in Target 'myproject.development.local' (project 'myproject'): script phase "[CP] Copy Pods Resources"
```

Downgrading to 3.1.0 fixes this issue

I have found similar issues reported in other libs (I am not using this lib, just sharing as an example of the same error I am facing with this lib) e.g. https://github.com/getsentry/sentry-react-native/issues/3599

### Reproduction

Already have a privacy manifest file thats target membership is the project and upgrade react-native-auth0 from `3.1.0` to `3.2.0`

### react-native-auth0 version

3.2.0

### React Native version

0.71.14

### Platform

iOS

### Platform version(s)

13

**Comment by bhandanyan-nomad** [comment]
Facing same issue since upgrading `react-native` to 0.71.19. Downgrading auth0 package to 3.1.0 from 3.2.0 fixes the issue

**Comment by quachdev** [comment]
Upgraded to RN `0.73.8`and using `3.2.0` facing same issue.

**Comment by andresfelipe3112** [comment]
Upgraded to RN 0.73.8and using 3.2.0 facing same issue.

**Comment by andresfelipe3112** [comment]
0.73.2 is good.

**Comment by desusai7** [comment]
Hi @DanielPritchard, @bhandanyan-nomad, @quachdev, @andresfelipe3112 

We've just released version `3.2.1` of the SDK with the fix for this issue, please upgrade to it and feel free to open back the issue, if this isn't resolved yet.
